### ISSUE TITLE
Duplicit (multiple) strings for translations

### ISSUE THREAD

**Issue by Miisa** [HabitRPG/habitrpg-ios#170]
Hello!

I started translating the iOS app and I noticed that some strings are there multiple times - To-Dos are there 4 times, Profile 3 times. Just like that, on their own. The problem is, we distinguish between the title of the column To-Dos and the To-Do tasks themselves. I looked into the Info about the strings and I cannot determine which is which so now I dunno which translation to use.

**Comment by vIiRuS** [comment]
Hi,

I can add notes to the different strings to provide context. I don't believe I can merge them though. XCode seems to want to keep them seperate for some reason.

I'm curious though. Why do you want to translate the column title different from the To-Do tasks?

**Comment by Miisa** [comment]
Yes, context would be great, thank you! 

And welp, there is equivalent to Dailies in Czech I had to work around it by adding descriptive element to the translation, which happens to be the equivalent for "task" which is also an equivalent for To-Do, so then I had to work around distinguishing To-Dos and tasks and got into this mess :D So basically, now there is the "To-Do" column that contains "tasks from the To-Do column". It was either that or using a descriptive phrase which would be very long.
